data/abilities.ts

```diff
@@ -4652,20 +4828,20 @@ export const Abilities: {[abilityid: string]: AbilityData} = {
 				return null;
 			}
 		},
-		isBreakable: true,
+		flags: {breakable: 1},
 		name: "Telepathy",
 		rating: 0,
 		num: 140,
 	},
 	teraformzero: {
 		onAfterTerastallization(pokemon) {
-			if (pokemon.baseSpecies.name !== 'Terapagos-Stellar') return;
```

Comment:
this should also be undeleted (also the tera shell check should probably be changed to use baseSpecies)